exercises/rotational-cipher/example.rs

```diff
@@ -0,0 +1,16 @@
+pub fn rotate(text: & 'static str, shift_key: u8) -> String {
```

Comment:
This might be OK since this is the example and won't be served to students, but I wouldn't want to restrict this function to only being able to take `&'static str`. I would expect it to be able to just take `&str`. So, if you remove `'static`, does the function still work? If not, why not?